packages/yum/groonga.spec.in

```diff
@@ -47,7 +49,7 @@ on relational data model.
 Summary:	Runtime libraries for Groonga
 Group:		System Environment/Libraries
 License:	LGPLv2 and (MIT or GPLv2)
-Requires:	arrow-libs >= @APACHE_ARROW_VERSION@, arrow-libs < @APACHE_ARROW_NEXT_MAJOR_VERSION@.0.0
+Requires:	%{_arrow_libs} >= @APACHE_ARROW_VERSION@, %{_arrow_libs} < @APACHE_ARROW_NEXT_MAJOR_VERSION@.0.0
```

Comment:
We can just write like `Requires: arrow@APACHE_ARROW_MAJOR_VERSION@-libs` because package name includes major version information.